probe-rs/src/config/flash_algorithm.rs

```diff
@@ -152,10 +153,16 @@ where
 }
 
 impl RawFlashAlgorithm {
-    const FLASH_BLOB_HEADER_SIZE: u32 = 8 * 4;
     const FLASH_ALGO_STACK_SIZE: u32 = 512;
     const FLASH_ALGO_STACK_DECREMENT: u32 = 64;
-    const FLASH_BLOB_HEADER: [u32; Self::FLASH_BLOB_HEADER_SIZE as usize / 4] = [
+
+    // Header for RISCV Flash Algorithms
+    const RISCV_FLASH_BLOB_HEADER: [u32; 2] = [
+        0b000000000001_00000_000_00000_1110011, // ebreak
```

Comment:
I know why you didn't format it like clippy wants, but maybe we should disable the lints then?